dockers/docker-fpm-frr/supervisord.conf

```diff
@@ -0,0 +1,59 @@
+[supervisord]
+logfile_maxbytes=1MB
+logfile_backups=2
+nodaemon=true
+
+[program:start.sh]
+command=/usr/bin/start.sh
+priority=1
+autostart=true
+autorestart=false
+startsecs=0
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:bgpcfgd]
+command=/usr/bin/bgpcfgd
+priority=2
+autostart=false
+autorestart=false
+startsecs=0
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:rsyslogd]
+command=/usr/sbin/rsyslogd -n
+priority=3
+autostart=false
+autorestart=false
+startsecs=0
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:zebra]
+command=/usr/lib/frr/zebra -A 127.0.0.1
```

Comment:
docker-fpm-frr/daemons has `zebra_options="  -A 127.0.0.1 -s 90000000 -M fpm"`,    is `-A 127.0.0.1` here necessary?

How does frr daemons config work with supervisord?

Reply:
fixed.